patches/third_party/devtools-frontend/src/front_end-models-extensions-ExtensionServer.ts.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/front_end/models/extensions/ExtensionServer.ts b/front_end/models/extensions/ExtensionServer.ts
+index 89925c919d28f25fe35a9c66f1e4b486be48d6cf..6e02337e082ce605ceca8267343793747e277480 100644
+--- a/front_end/models/extensions/ExtensionServer.ts
++++ b/front_end/models/extensions/ExtensionServer.ts
+@@ -1331,6 +1331,8 @@ export class ExtensionServer extends Common.ObjectWrapper.ObjectWrapper<EventTyp
+     if (kAllowedOrigins.includes(parsedURL.origin)) {
+       return true;
+     }
++    // @ts-ignore
++    if (window.DevToolsAPI && window.DevToolsAPI.getEnabledOnChromeUrls && window.DevToolsAPI.getEnabledOnChromeUrls() && parsedURL.protocol === 'chrome:') { return true; }
```

Comment:
optional nit:

```suggestion
+    if (window.DevToolsAPI?.getEnabledOnChromeUrls?.() && parsedURL.protocol === 'chrome:') { return true; }
```

Reply:
Sure!
I've just tried to be consistent with what is already there https://chromium-review.googlesource.com/c/devtools/devtools-frontend/+/3428454/5/front_end/models/extensions/ExtensionServer.ts#1110